src/Disks/DiskSelector.cpp

```diff
@@ -60,14 +60,17 @@ void DiskSelector::initialize(
     }
     if (!has_default_disk)
     {
-        disks.emplace(
-            DEFAULT_DISK_NAME, std::make_shared<DiskLocal>(DEFAULT_DISK_NAME, context->getPath(), 0, context, config, config_prefix));
+        auto disk = std::make_shared<DiskLocal>(DEFAULT_DISK_NAME, context->getPath(), 0, context, config, config_prefix);
+        disk->startup(/*skip_access_check=*/ true);
```

Comment:
Why we are skipping access checks here?

Reply:
We didn't run `startup` for this disk at all previously (and so access check), so I decided to keep old behaviour just in case. But I can enable it and we can see if it will break something